src/SDKs/MarketplaceOrdering/MarketplaceOrdering.Tests/MarketplaceOrdering.Tests.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />  
+  <PropertyGroup>
+    <PackageId>MarketplaceOrdering.Tests</PackageId>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <Description>MarketplaceOrdering.Tests Class Library</Description>
+    <Authors>Microsoft Corporation</Authors>
+    <AssemblyName>MarketplaceOrdering.Tests</AssemblyName>    
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
```

Comment:
Any reasons you need this ItemGroup? If none, Please remove

Reply:
There is no reason, deleting this.